model-optimizer/mo/ops/op.py

```diff
@@ -445,7 +445,7 @@ def get_nhwc_to_nchw_permutation(dims_number: int):
             # Exclude 3D shapes from permutation process: identity permutation
             perm = list(range(0, dims_number))
         inv = PermuteAttrs.get_inverse_permutation(perm)
-        return PermuteAttrs.Permutation(perm=np.array(perm), inv=np.array(inv))
+        return PermuteAttrs.Permutation(perm=np.array(perm, dtype=np.int32), inv=np.array(inv, dtype=np.int32))
```

Comment:
Why this change is needed? If we some case when `perm` contains floating point values then we should find why they appear and fix that instead of hiding the issue here.

Reply:
Permutation is created here and in case empty list the array is of float64 type. It didn't affect early because such permutations  were created for non-parameter scalar nodes. For TF2 While we have two auxiliary body parameters nodes that are scalars and this is why error is happening.

Reply:
Lets use `int64_array(xxx)` instead of using `np.array(xxx, dtype=np.int32)`

Reply:
I don't see that my comment was applied.

Reply:
Fixed.